src/libos/src/signal/do_sigreturn.rs

```diff
@@ -309,6 +323,15 @@ impl Stack {
         Ok(Stack { pointer, bottom })
     }
 
+    /// Get the minimum memory range which is the superset of the stack
+    pub fn range(&self) -> VMRange {
+        VMRange::new(
+            align_down(self.bottom, PAGE_SIZE),
+            align_up(self.pointer, PAGE_SIZE),
+        )
+        .unwrap()
+    }
```

Comment:
In accordance with the comments above, the `Stack` should not have such a method.